docs/commands/clip.md

```diff
@@ -0,0 +1,30 @@
+# clip (Windows Only)
+
+use clip to redirect the output of a command to windows clipboard (copy)
+
+
+# Syntax
+```cmd
+<command> | clip
+```
+```cmd
+clip < <file to copy from>
+```
+
+# Parameters
```

Comment:
The `clip` command doesn't have any parameters.  You can remove this section, or just say that it works with parameters.

Reply:
Removed the section